provider-gui/services/contract/config.ts

```diff
@@ -0,0 +1,49 @@
+import {
+    DatabaseTypes,
+    EnvironmentTypes,
+    NetworkNames,
+    NetworkPairTypeSchema,
+    ProsopoConfigSchema,
+} from '@prosopo/types'
+import { getLogLevel } from '@prosopo/common'
+
+const logLevel = getLogLevel()
+export const getConfig = (environment: EnvironmentTypes, network: NetworkNames) => {
+    return ProsopoConfigSchema.parse({
+        logLevel,
+        defaultEnvironment: environment,
+        defaultNetwork: network,
+        account: {
+            password: '',
+            address: 'currentAccount',
+        },
+        database: {
+            development: { type: DatabaseTypes.enum.mongoMemory, endpoint: '', dbname: 'prosopo_test', authSource: '' },
+        },
+        server: {
```

Comment:
This is for the image server. You shouldn't need to declare it.

Reply:
Removed